### PR TITLE
MP Open API Configured

### PR DESCRIPTION
**Pull request by jjvilleg**

### CODE REVIEW

File: src/main/java/com/ibm/hybrid/cloud/sample/stocktrader/tradehistory/rest/Trades.java [hunk 1]
```diff
@@ -95,6 +147,18 @@ public String getROI(
     @Path("/shares/{owner}/{symbol}")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
+    @APIResponses(value = {
+        @APIResponse(
+            responseCode = "404",
+            description = "The mongo database cannot be fround. ",
```
Comment (yeekangc) [inline src/main/java/com/ibm/hybrid/cloud/sample/stocktrader/tradehistory/rest/Trades.java]:
```suggestion
            description = "The Mongo database cannot be found. ",
```

File: src/main/java/com/ibm/hybrid/cloud/sample/stocktrader/tradehistory/rest/Trades.java [hunk 2]
```diff
@@ -107,6 +171,18 @@ public String getCurrentShares(
     @Path("/shares/{owner}")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
+    @APIResponses(value = {
+        @APIResponse(
+            responseCode = "404",
+            description = "The mongo database cannot be fround. ",
```
Comment (yeekangc) [inline src/main/java/com/ibm/hybrid/cloud/sample/stocktrader/tradehistory/rest/Trades.java]:
```suggestion
            description = "The Mongo database cannot be found. ",
```

File: src/main/java/com/ibm/hybrid/cloud/sample/stocktrader/tradehistory/rest/Trades.java [hunk 3]
```diff
@@ -117,6 +193,18 @@ public String getPortfolioShares(@Parameter(description="Owner name", required =
     @Path("/notional/{owner}")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
+    @APIResponses(value = {
+        @APIResponse(
+            responseCode = "404",
+            description = "The mongo database cannot be fround. ",
```
Comment (yeekangc) [inline src/main/java/com/ibm/hybrid/cloud/sample/stocktrader/tradehistory/rest/Trades.java]:
```suggestion
            description = "The Mongo database cannot be found. ",
```

File: src/main/java/com/ibm/hybrid/cloud/sample/stocktrader/tradehistory/rest/Trades.java [hunk 4]
```diff
@@ -127,6 +215,18 @@ public String getNotional(
     @Path("/returns/{owner}")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
+    @APIResponses(value = {
+        @APIResponse(
+            responseCode = "404",
+            description = "The mongo database cannot be fround. ",
```
Comment (yeekangc) [inline src/main/java/com/ibm/hybrid/cloud/sample/stocktrader/tradehistory/rest/Trades.java]:
```suggestion
            description = "The Mongo database cannot be found. ",
```

**Comment by jjvilleg** [comment]
Nice Catch! My keyboard has been acting up  😅

**Pull request merged**